### DISCUSSION TITLE
Is there a quick way to make ssh / telnet connection?

### DISCUSSION THREAD

**Discussion by ManchiSoul** [Eugeny/tabby#4976] [Q&A] [answered]
Hello

When I press + button I got default cmd window and if I press on that two window icon next to + icon. I will go to profile and I will have to create a profile for the new ssh / telnet connection that I want just quick access for it

Is there a quick way to make ssh or telnet connections without making a whole profile for it?

Regards.

**Comment by Eugeny** [comment, marked as answer]
Yes - if you press the second button (profile selector), you can just type in an address and press Enter.

**Reply by BallisticCrux** [reply]
how do I tell tabby it to try using a telnet session instead of defaulting to ssh using this method?

**Comment by lawweiliang** [comment]
@Breetlez 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
